packages/config-vite-base/package.json

```diff
@@ -10,6 +10,19 @@
   "scripts": {
     "build": "tsup"
   },
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
+    },
+    "./setup": "./src/setup.ts"
```

Comment:
Exporting the setup file as a direct source file for other vitests to use as a setup file.